### PR TITLE
🌐 Add Spanish translation for `docs/es/docs/history-design-future.md`

### PR DESCRIPTION
**Pull request by pablocm83**
- Version >= 0.108.0

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit 048fe3bd1e143683bf2c8aa0454576bc47d0ea6f at: https://20610833.fastapitiangolo.pages.dev

File: docs/es/docs/history-design-future.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+Entonces, incluso antes de comenzar a codificar **FastAPI**, pasé varios meses estudiando las especificaciones de OpenAPI, JSON Schema, OAuth2, etc. Entendiendo su relación, superposición y diferencias.
+
+## Diseño
+
+Luego dediqué algún tiempo a diseñar la "API" de desarrollador que quería tener como usuario (como desarrollador que usa FastAPI).
+
+Probé varias ideas en los editores de Python más populares: PyCharm, VS Code, editores basados en Jedi.
+
+Según la última <a href="https://www.jetbrains.com/research/python-developers-survey-2018/#development-tools" class="external-link" target="_blank">Encuesta para Desarrolladores de Python< /a>, que cubre alrededor del 80% de los usuarios.
```
Comment (JavierSanchezCastro) [inline docs/es/docs/history-design-future.md:47]: Etiqueta mal cerrada, fallo en visualización

File: docs/es/docs/history-design-future.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+Significa que **FastAPI** fue probado específicamente con los editores utilizados por el 80% de los desarrolladores de Python. Y como la mayoría de los demás editores tienden a trabajar de manera similar, todos sus beneficios deberían funcionar para prácticamente todos los editores.
+
+De esa manera podría encontrar las mejores formas de reducir la duplicación de código tanto como sea posible, completarlo en todas partes, verificar tipos y errores, etc.
+
+Todo de una manera que brindó la mejor experiencia de desarrollo para todos los desarrolladores.
+
+## Requisitos
+
+Después de probar varias alternativas, decidí que iba a utilizar <a href="https://pydantic-docs.helpmanual.io/" class="external-link" target="_blank">**Pydantic**< /a> por sus ventajas.
```
Comment (JavierSanchezCastro) [inline docs/es/docs/history-design-future.md:57]: Tag mal cerrado

File: docs/es/docs/history-design-future.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+Durante el desarrollo, también contribuí a <a href="https://www.starlette.io/" class="external-link" target="_blank">**Starlette**</a>, el otro requisito clave.
+
+## Desarrollo
+
+Cuando comencé a crear **FastAPI**, la mayoría de las piezas ya estaban en su lugar, el diseño estaba definido, los requisitos y las herramientas estaban listos y el conocimiento sobre los estándares y especificaciones era claro y actualizado.
+
+## Futuro
+
+en este punto, ya está claro que **FastAPI** con sus ideas está siendo útil para muchas personas.
```
Comment (JavierSanchezCastro) [inline docs/es/docs/history-design-future.md:69]: Mayúscula

**Comment by tiangolo** [comment]
📝 Docs preview for commit cacf642c340fdbd3d66a9fae65fe2419d61ec114 at: https://a0e4d157.fastapitiangolo.pages.dev

File: docs/es/docs/history-design-future.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
+# Historia, Diseño y Futuro
+
+Hace algún tiempo, <a href="https://github.com/tiangolo/fastapi/issues/3#issuecomment-454956920" class="external-link" target="_blank">a **FastAPI** un usuario preguntó</a>:
+
+> ¿Cuál es la historia de este proyecto? Parece haber llegado de la nada a ser asombroso en unas pocas semanas [...]
+
+Aquí explico.
```
Comment (JavierSanchezCastro) [inline docs/es/docs/history-design-future.md:7]: Aquí te explico.

(Me parece que suena mejor)

**Comment by tiangolo** [comment]
📝 Docs preview for commit ec9f748ac7d606583b6806fca8f153af69bd9391 at: https://fdbc941a.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit ad46a49ebeafb33640eb7b07610ddf594077dae4 at: https://7af0e3e3.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
This was covered in another PR, so I'll close this one now, thanks for the effort! :cake: :coffee:

**Pull request closed without merging**